sdk/storage/storage-blob/samples/typescript/azureAdAuth.ts

```diff
@@ -1,5 +1,20 @@
 /*
- Setup: Enter your Azure Active Directory credentials as described in main()
+  ONLY AVAILABLE IN NODE.JS RUNTIME
+
+  Setup :
+    - Reference - [Authorize access to blobs and queues with Azure Active Directory from a client application](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth-aad-app)
```

Comment:
Do the Markdown-style links actually get rendered correctly somewhere? If not, maybe it's clearer to just paste the link after the text without the bracket syntax.

Reply:
Okay. I'll remove the brackets and keep the links only.